src/components/AddButton/AddButton.scss

```diff
@@ -0,0 +1,48 @@
+@import "src/styles/variables/_variables";
+@import "src/styles/Icon";
+
+$borderColor: $linkColor;
+
+.addButton {
```

Comment:
(Minor : Coding Style ) : I saw few places where we use "-"  dash separator for class name. Here I see camel case.
My preferred coding style is dash separator.  So `.add-button` instead of `.addButton` 
Better choose one or the other and stay consistent about it!